### DISCUSSION TITLE
Multiple Stripe webhooks pointing to 1 endpoint

### DISCUSSION THREAD

**Discussion by theendcomplete** [pay-rails/pay#611] [Q&A] [answered]
Hi!
Is there any possibility to make Pay use 2 different Stripe webhook secrets at the same time?

It seems that `Connect` webhooks send only connected account events without direct.

So it is required to setup 2 different Stripe webhooks.

The gem can process `connect` events, and it works with Stripe CLI redirecting events to local machine.

But for production and staging env it is required to setup 2 signatures at the same time. Is it possible? 

The project uses ENV variables as it is deployed to Heroku.

upd: It seems that the gem can work with multiple signatures out of the box, but there is no way to set up these signatures via env variables out of the box.

**Comment by excid3** [comment, marked as answer]
Yep, put the signing secret in your Rails credentials as an array.

**Reply by theendcomplete** [reply]
thank you!
